website/docs/cloud-docs/projects/index.mdx

```diff
@@ -29,9 +29,9 @@ departments, subsidiaries, or technical teams.
 ## Default Project
 
 Every workspace must belong to exactly one project. By default, all workspaces
-belong to an organization's **Default Project**. You can rename the default
-project, but you cannot delete it. You can specify a workspace's project at the
-time of creation and move it to a different project later. 
+belong to an organization's **Default Project**. You can rename and change the
+default project, but you cannot delete it. You can specify a workspace's project
+at the time of creation and move it to a different project later.
```

Comment:
Instead of suggestions on multiple lines, (which I always find hard to read), here is my suggestion:
"
You can rename the default project or set another project as the default, but you cannot delete an organization's default project. When creating a workspace, you can specify which project it belongs to, and you can always move it to a different project later. If you are creating a workspace and your organization does not have a default project, you must specify which project your workspace belongs to. 
"

How does that feel?